internal/booking/timeedit/room.go

```diff
@@ -28,3 +28,23 @@ func (rs rooms) nameFromId(id string) (string, error) {
 	}
 	return "", fmt.Errorf("no such room")
 }
+
+func (rs rooms) removeAt(i int) rooms {
+	return append(rs[:i], rs[i+1:]...)
+}
+
+func (rs rooms) remove(room room) rooms {
+	for i, r := range rs {
+		if r == room {
+			return rs.removeAt(i)
+		}
+	}
+	return rs
+}
+
+func (rs rooms) removeMany(rooms rooms) rooms {
+	for _, n := range rooms {
+		rs = rs.remove(n)
+	}
+	return rs
+}
```

Comment:
Looks like this code is dead?

Reply:
Nope it's used [here](https://github.com/sidusIO/BooGroCha/pull/50/files#diff-4caae089efd20f4253f987e704f0133aR287) in `getRooms()`

Reply:
Not that I could find